examples/todo-typescript/src/client/SignupPage.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import SignupForm from '@wasp/auth/forms/Signup';
```

Comment:
I recommend avoiding default exports, check https://github.com/wasp-lang/wasp/issues/546 for details.

Reply:
isn't Wasp itself exporting the form as a default export?

Reply:
Indeed it is :sweat_smile:
My bad!